CommentAgent.py

```diff
@@ -360,12 +254,15 @@ def review_pr(self) -> bool:
         for _ in range(3):
             message_format = GPTsettings.MESSAGE_FORMAT
             self.GPT.add_message("user", message_format)
-            message_response = self.GPT.get_response(max_tokens=2500)
+            message_response = self.GPT.get_response(
+                max_tokens=4096
+            )  # Max length of response is 4096 (max allowed for responses by openai)
             try:
                 logging.info(message_response)
                 response_parser = ResponseParser(message_response)
                 body, event, comments = response_parser.get_body_event_and_comments()
                 self.pr.create_review(body=body, event=event, comments=comments)
+                print(self.GPT)
```

Comment:
This print statement could be replaced with a logging call for better control of the output, consistency, and logging levels.
```suggestion
    logging.info(str(self.GPT))
```